scripts/ms

```diff
@@ -77,7 +87,7 @@ def init(opts):
                  "out_dir": out_dir,
                  "out_state": out_state,
                  "state": state_file,
-                 "annual": opts.annual,
+                 "time_grouper": opts.time_grouper,
```

Comment:
I think it would be nice to have this pulled out in a lower line to be something like:
```
if opts.time_grouper:
    conf['time_grouper'] = opts.time_grouper
```
That way we could specify the `time_grouper` in the configuration file, or override it on the command line.